### PR TITLE
fix(collector-options-forms): modify request params of getPluginMetadata

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA 
 & fix css

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <template>
-    <p-data-loader class="collector-options-form"
+    <p-data-loader class="collector-options-form-container"
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:2]: suggestion: Oh this is wrong convention.

https://cloudforet.io/ko/docs/developers/frontend/coding_convention/
image

Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorOptionsForm.vue:2]: Oops, that's only mirinae's convention. 😅

But in CollectorOptionsSection component, it uses the class name 'collector-options-form' to add css in style block.
So if you want to change the class name, please change there too!

**Review by WANZARGEN — COMMENTED** [review]
Good! I left a suggestion!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Review by WANZARGEN — APPROVED** [review]

**Pull request merged**